src/style.css

```
@@ -0,0 +1,130 @@
+.page-content {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 0 16px;
+
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  color: #060b35;
+  text-align: center;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 37px;
+}
+
+.card__product-name {
+  margin: 0 16px 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  text-align: left;
+}
+
+.card__product-reviews,
+.card__product-price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__product-code,
+.card__product-reviews {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+}
+
+.card__product-code {
+  margin: 0 16px 16px;
+
+  color: #616070;
+
+  text-align: left;
+}
+
+.card__product-reviews {
+  margin: 0 16px 24px;
+}
+
+.card__reviews {
+  margin: 0;
+}
+
+.card__product-price {
+  margin: 0 16px 16px;
+
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+.card__price {
+  display: inline-block;
+  margin: 0;
+}
+
+.card__price--bold-sum {
+  font-weight: bold;
+  font-size: 16px;
+  white-space: nowrap;
+}
+
+.card__link {
+  display: block;
+  padding: 11px 0;
+  margin: 0 16px;
+
+  color: #fff;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  border: 1px solid #00acdc;
+  background-color: #00acdc;
+  border-radius: 5px;
+}
+
+.card__link:hover {
+  color: #00acdc;
+
+  background: #fff;
+}
+
+.stars {
+  display: inline-flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
```

в html кроме stars--4 ниче нету, значит остальные классы не нужны?
в остальном четко!
```suggestion
.stars--4 :nth-child(-n+4) {
```

---

we need it: when a user clicks on the stars, with javascript we will change the class and the necessary values will be applied

---

@oleksandr-haidysh тут ты был не прав, в 3 пункте чеклиста сказано, что нужны все классы